app/src/androidTest/java/com/w3engineers/unicef/telemesh/ui/createuser/CreateUserViewModelTest.java

```diff
@@ -74,8 +74,9 @@ public void isNameValid_true_setValidString() {
     @Test
     public void storeData_true_setValidData() {
         String firstName = "Daniel";
+        String lastName = "Craig";
         String password = "mesh_123";
```

Comment:
is there any security concern on static password set here @W3-Tariqul?

Reply:
No. It is the test account and password, and it is for Unit test.